api/controllers/zetter.js

```diff
@@ -18,6 +18,21 @@ const createTweet = (req, res) => {
     res.status(201).json(newTweet);
 };
 
+const getProfile = (req, res) => {
+    const user = users.find((user) => user.username === req.user.username);
+
+    res.json(user);
```

Comment:
```suggestion
    res.json(req.user);
```
ミドルウェアですでにデータベースから詳細なユーザー情報を取得しているので、再取得する必要はないよ